api/lib/opentelemetry/trace/tracer.rb

```diff
@@ -85,14 +85,9 @@ def start_root_span(name, attributes: nil, links: nil, start_timestamp: nil, kin
       #
       # Parent context can be either passed explicitly, or inferred from currently activated span.
       #
-      # @param [optional Span] with_parent Explicitly managed parent Span, overrides
-      #   +with_parent_context+.
-      # @param [optional Context] with_parent_context Explicitly managed. Overridden by
-      #   +with_parent+.
-      #
       # @return [Span]
-      def start_span(name, with_parent: nil, with_parent_context: nil, attributes: nil, links: nil, start_timestamp: nil, kind: nil)
-        span_context = with_parent&.context || active_span_context(with_parent_context)
+      def start_span(name, with_parent: nil, attributes: nil, links: nil, start_timestamp: nil, kind: nil)
+        span_context = active_span_context(with_parent)
```

Comment:
Can we make `active_span_context` private or `@api private`? Or is it required by the spec? (We don’t use it anywhere except in the API and SDK `Tracer` classes.)

Reply:
I made it private. I think we can remove this method completely by simplifying the current span situation. I'll look into opening a separate PR that does that tomorrow.